srcpkgs/tuimoji/template

```diff
@@ -1,11 +1,12 @@
 # Template file for 'tuimoji'
 pkgname=tuimoji
 version=1.0.0
-revision=7
+revision=8
 build_style=python3-module
-pycompile_module="tuimoji"
+#pycompile_module="tuimoji"
 hostmakedepends="python3-setuptools"
 depends="python3-urwid xclip python3-setuptools"
+checkdepends="python3-pip"
```

Comment:
this is probably wrong, something saying "module pip not found" generally means something else wasn't found and needs to be added, so it was trying and failing to install it with pip

Reply:
```
=> tuimoji-1.0.0_8: running do_check ...
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
/usr/lib/python3.12/site-packages/setuptools/command/test.py:193: _DeprecatedInstaller: setuptools.installer and fetch_build_eggs are deprecated.
!!

        ********************************************************************************
        Requirements should be satisfied by a PEP 517 installer.
        If you are using pip, you can try `pip install --use-pep517`.
        ********************************************************************************

!!
  ir_d = dist.fetch_build_eggs(dist.install_requires)
/usr/bin/python3: No module named pip
error: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmpqi9nfp5u', '--quiet', 'urwid']' returned non-zero exit status 1.
=> ERROR: tuimoji-1.0.0_8: do_check: '${make_check_pre} python3 setup.py ${make_check_target} ${make_check_args}' exited with 1
=> ERROR:   in do_check() at common/build-style/python3-module.sh:28

```
it want to install `wheel` using pip, while `checkdepends="python3-wheel'`

Reply:
I realized that this project does not have any test

Reply:
ok i figured out thanks

Reply:
Just a question: test passes without `python3-wheel` leave a warning of missing `wheel`
so should I be added in `checkdepends=`